qiling/extensions/r2/r2.py

```diff
@@ -102,14 +103,35 @@ class SymbolBind(str, Enum):
 
 
 class R2:
-    def __init__(self, ql: Qiling):
+    def __init__(self, ql: Qiling, baseaddr=(1 << 64) - 1, loadaddr=0):
```

Comment:
a base address of 0xffffffffffffffff ? are you sure..?

Reply:
I guess it's a default value hardcoded in r2 when calling `r_core_bin_load` but I am not sure. I have debugged functions `r_core_bin_load` and `r_core_file_open` to see the exact value of `baseaddr` and `loadaddr`. Maybe I should left some comments about them.

See https://github.com/radareorg/radare2/blob/master/libr/main/radare2.c#L427

Reply:
> a base address of 0xffffffffffffffff ? are you sure..?

Yes, that's the default value indeed.

Reply:
So `-1` is a special case there..?
I don't see any other explanation for picking this value as default.